pkg/scheduler/scheduling_queue.go

```diff
@@ -558,6 +575,19 @@ func (sq *SchedulingQueue) tryAllocate(ctx *partitionSchedulingContext) *schedul
 	return nil
 }
 
+func (sq *SchedulingQueue) getQueueOutstandingRequests(total *outstandingRequests) {
```

Comment:
Suggest to rename to `getAllocatableOutstandingRequests`

Reply:
See above, not sure that this is allocatable